templates/ecmonsen-emorypipeline-ci.yaml

```diff
@@ -0,0 +1,140 @@
+Description: Setup service account and roles for Emory pipeline lambda
```

Comment:
this same file is duplicated in both sandbox-provisioner and scicomp-provisioner.  would it make sense to move this to `Sage-Bionetworks/ecmonsen-emorypipeline-lambda-code` repro?  This would allow you to  deploy it to your public bucket and then shared to sandbox-provisioner and scicomp-provisioner?